### ISSUE TITLE
EditorConfig: wrong validation message for rule which allows only lower symbols in name

### ISSUE THREAD

**Issue by v-margko** [dotnet/roslyn#25959]
**Version Used**: 
Microsoft Visual Studio Enterprise 2017 Int Preview Version 15.7.0 Preview 3.0 [27603.0.d15.7]

**Steps to Reproduce**:

1.	Create new project Class Library (.NET Core)
2.	Add .editorconfig to project:
```ini
[*]
dotnet_naming_rule.test.symbols   = private_symbols
dotnet_naming_symbols.private_symbols.applicable_kinds = property,method,field,event,delegate

dotnet_naming_rule.test.style    = lower_symbols
dotnet_naming_style.lower_symbols.capitalization = all_lower

dotnet_naming_rule.test.severity = suggestion
```
3.	Open existing *.cs class
4.	Rename any method to name with upper symbols only
5.	Save the file and reopen

**Expected Behavior**: "These words **should** contain lower case characters" notification should be displayed

**Actual Behavior**: Validation for the method above is displayed with a message: These words cannot contain lower case characters
![image](https://user-images.githubusercontent.com/9797472/38388488-371e7628-38d0-11e8-90e7-9f355c7acb9c.png)

**Comment by sharwell** [comment]
This was fixed in #23642 for the 15.8 release

**Comment by v-rodudi** [comment]
Verified on VS 15.9.0 Preview 2.0 [28029.0.d15.9]. Issue is not reproduced.
